docs/tools/kit/index.md

```diff
@@ -377,11 +377,12 @@ function RandomValues() {
 
 #### Notes:
 
-* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and transaction execution.
-* The value returned for identical calls within the same block will be identical.
-* This hook is designed for simple use cases that **don’t require a [commit-reveal scheme]**, such as randomized UIs.
-* For **more advanced use cases** that **do** require a [commit-reveal scheme], Flow provides built-in support for that pattern.
-* Values are **deterministic**: if a transaction fails and retries, the same random value will be regenerated.
+* Randomness is generated using Flow’s **on-chain `revertibleRandom`**, producing pseudorandom values tied to block and script execution.
```

Comment:
there are no transactions in this case, it's all script based (under the hood, this makes a difference in how randomness works).